packages/repeater/src/api/HttpRepeatersManager.spec.ts

```diff
@@ -0,0 +1,77 @@
+import 'reflect-metadata';
+import {
+  CreateRepeaterRequest,
+  DeleteRepeaterRequest,
+  ListRepeatersRequest
+} from './commands';
+import { HttpRepeatersManager } from './HttpRepeatersManager';
+import { RepeatersManager } from './RepeatersManager';
+import { CommandDispatcher } from '@secbox/core';
+import { anyOfClass, instance, mock, reset, verify, when } from 'ts-mockito';
+
+describe('HttpRepeatersManager', () => {
+  const MockedCommandDispatcher = mock<CommandDispatcher>();
+  let manager!: RepeatersManager;
+
+  beforeEach(() => {
+    manager = new HttpRepeatersManager(instance(MockedCommandDispatcher));
+  });
+
+  afterEach(() => reset(MockedCommandDispatcher));
+
+  describe('createRepeater', () => {
+    it('should create repeater', async () => {
+      when(
+        MockedCommandDispatcher.execute(anyOfClass(CreateRepeaterRequest))
+      ).thenResolve();
+
+      when(
+        MockedCommandDispatcher.execute(anyOfClass(ListRepeatersRequest))
+      ).thenResolve([
+        { name: 'bar', id: '142' },
+        {
+          name: 'foo',
+          id: '42'
+        }
+      ]);
+
+      const result = await manager.createRepeater({ name: 'foo' });
+
+      verify(
+        MockedCommandDispatcher.execute(anyOfClass(CreateRepeaterRequest))
+      ).once();
+      verify(
+        MockedCommandDispatcher.execute(anyOfClass(ListRepeatersRequest))
+      ).once();
+      expect(result.repeaterId).toBe('42');
+    });
+
+    it('should throw an error if cannot find created repeater', async () => {
+      when(
+        MockedCommandDispatcher.execute(anyOfClass(CreateRepeaterRequest))
+      ).thenResolve();
+      when(
+        MockedCommandDispatcher.execute(anyOfClass(ListRepeatersRequest))
+      ).thenResolve(null);
+
+      const res = manager.createRepeater({ name: 'foo' });
+
+      await expect(res).rejects.toThrow('Cannot find created repeater id');
+    });
+  });
+
+  describe('deleteRepeater', () => {
+    it('should remove repeater', async () => {
+      when(
+        MockedCommandDispatcher.execute(anyOfClass(DeleteRepeaterRequest))
+      ).thenResolve();
+
+      const res = manager.deleteRepeater('fooId');
+
+      verify(
+        MockedCommandDispatcher.execute(anyOfClass(DeleteRepeaterRequest))
+      ).once();
+      await expect(res).resolves.not.toThrow();
```

Comment:
You can get rid of this redundant assertion. If a promise is rejected, the test will fall as well

```suggestion
      const res = await manager.deleteRepeater('fooId');

      verify(
        MockedCommandDispatcher.execute(anyOfClass(DeleteRepeaterRequest))
      ).once();
```